### PR TITLE
completed javascript-modal. katiefugate.

### PR DESCRIPTION
**Pull request by katiefugate**
Screen Shot 2021-03-23 at 3 09 04 PM
Screen Shot 2021-03-23 at 3 09 16 PM

### CODE REVIEW

File: javascript-modal/style.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.open {
+  background-color: rgb(71, 55, 223);
+  color: white;
+  padding: 5px 25px;
+  border: 1px solid rgb(71, 55, 223);
+  border-radius: 5px;
+  font-size: 20px;
+  position: absolute;
```
Comment (codingcodymiller) [inline javascript-modal/style.css:12]: This button should be in the normal document flow, it shouldn't be absolute positioned. You could absolute or fixed position the modal, that is the content meant to sit on a layer above normal content.

File: javascript-modal/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="style.css" rel="stylesheet">
+  <title>JavaScript Modal</title>
+</head>
+<body>
+<button class="open">Open Modal</button>
+<div class="modalContainer">
+<dialog class="modal">
```
Comment (codingcodymiller) [inline javascript-modal/index.html:13]: Very cool usage of the `dialog` element!! Unfortunately, the `dialog` element still isn't well-incorporated across all browsers, so you won't want to make use of it just yet. I recommend looking back to the `fixed` portion of the `css-positioning` exercise for some tips on how to accomplish this without a `dialog` element

https://caniuse.com/dialog

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! Please see my notes

**Comment by katiefugate** [comment]
Screen Shot 2021-03-23 at 5 01 51 PM
Screen Shot 2021-03-23 at 5 02 02 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**